core/src/main/java/com/sequenceiq/cloudbreak/job/existingstackfix/ExistingStackFixerJobService.java

```diff
@@ -0,0 +1,89 @@
+package com.sequenceiq.cloudbreak.job.existingstackfix;
+
+import javax.inject.Inject;
+
+import org.quartz.JobBuilder;
+import org.quartz.JobDataMap;
+import org.quartz.JobDetail;
+import org.quartz.JobKey;
+import org.quartz.Scheduler;
+import org.quartz.SchedulerException;
+import org.quartz.SimpleScheduleBuilder;
+import org.quartz.Trigger;
+import org.quartz.TriggerBuilder;
+import org.quartz.impl.matchers.GroupMatcher;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ExistingStackFixerJobService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ExistingStackFixerJobService.class);
+
+    private static final String JOB_GROUP = "existing-stack-fixer-jobs";
+
+    private static final String TRIGGER_GROUP = "existing-stack-fixer-triggers";
+
+    private static final String LOCAL_ID = "localId";
+
+    private static final String REMOTE_RESOURCE_CRN = "remoteResourceCrn";
```

Comment:
I see these multiple times in our code. I think it worth it to move to somewhere common instead of duplicating them again